pynetgear/__init__.py

```diff
@@ -151,8 +152,8 @@ def get_attached_devices_2(self):
             return None
 
         success, devices_node = _find_node(
```

Comment:
You missed at least one _find_node call, please do ctrl+F and check the calls.

Reply:
hmm. I only see 3 calls to `_find_node` and all of them are changed.

Reply:
Hum indeed need more sleep, sorry for that.